test/Sema/diag_ownership_lazy.swift

```diff
@@ -1,7 +1,19 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -verify-ignore-unknown
+
+// -verify-ignore-unknown is for:
+// <unknown>:0: error: unexpected note produced: 'self' declared here
```

Comment:
Actually, to avoid all this you could just give the properties a type, or make a different instance other than `self`

Reply:
A good point. There's no real reason to test with `= self` in this case since we're testing the lazy attribute/modifier itself. I will change this test!